File: isletimsistemlericalismasiJava-calismalarim/app/src/main/java/com/example/isletimsistemicalismasi/MainActivity.java
package com.example.isletimsistemicalismasi;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.app.AppCompatDelegate;

import android.os.Bundle;

import com.example.isletimsistemicalismasi.anaekran.entity.Const;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: SozlukKotlin/app/src/main/java/com/example/sozlukkotlin/MainActivity.kt
package com.example.sozlukkotlin

import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import androidx.navigation.findNavController
import com.example.sozlukkotlin.util.SessionManager

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val navController = findNavController(R.id.navHostFragment)
        val session = SessionManager(this)

        val currentDestination = navController.currentDestination?.id

        // Uygulama ilk açıldığında oturum kontrolü
        if (savedInstanceState == null) {
            if (session.isLoggedIn()) {
                navController.navigate(R.id.anaSayfaFragment)
            } else {
                navController.navigate(R.id.girisFragment)
            }
        }
    }
}
